packages/create-next-app/templates/default/pages/index.js

```diff
@@ -57,7 +58,13 @@ export default function Home() {
           rel="noopener noreferrer"
         >
           Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
+          <img
```

Comment:
This can use next/image

Reply:
I was thinking of adding next/image here but wasn't sure for the purpose of the default template did you want to include it? I can update to use if so.

Reply:
Yeah feel free to add it in a separate PR 👍